plugins/packs-integrations.js

```diff
@@ -9,25 +15,218 @@ function generateIntegrationData(allContent) {
   return packsData;
 }
 
-function generatePacksData(allContent) {
-  const packsData = allContent["docusaurus-plugin-content-docs"].default.loadedVersions[0].docs
-    .filter((doc) => {
-      return doc.frontMatter.type === "integration";
+function getReadMeMap(packValues) {
+  const map = new Map();
+  packValues.forEach((packValue) => {
```

Comment:
reduce can be used here.
side note: you should use map.set instead of assigning attributes on this map

Reply:
generateCustomData